pkg/cli/repl.go

```diff
@@ -39,6 +39,10 @@ var (
 	}
 
 	getSuggestions = []prompt.Suggest{
+		{
+			Text:        "acls",
+			Description: "Get all acls",
```

Comment:
"acls" should be capitalized here and below at line 471 for consistency with other text output

Reply:
Good catch, fixed